src/docs/deployment/android.md

```diff
@@ -235,6 +235,52 @@ Using the command line:
 For detailed instructions on publishing the release version of an app to the
 Google Play Store, see the [Google Play publishing documentation][play].
 
+## Building a release app bundle
+
+This section describes how to build a release app bundle. If you completed
+the signing steps in the previous section, the release bundle will be signed.
+
+From the command line:
+
+1. Enter `cd <app dir>`. (Replace `<app, dir>` with your application's directory.)
+1. Run `flutter build appbundle`. (Running `flutter build` defaults to a release build.)
+1. To generate a different variant of bundle, you can enter
```

Comment:
I don't quite understand this bit. Is this about build variants (https://developer.android.com/studio/run/#changing-variant) or choosing an ABI?